docs/_includes/generated/rest_v1_dispatcher.html

```diff
@@ -4128,3 +4128,116 @@
     </tr>
   </tbody>
 </table>
+<table class="table table-bordered">
+    <tbody>
+    <tr>
+        <td class="text-left" colspan="2"><h5><strong>/taskmanagers/:taskmanagerid/logs</strong></h5></td>
+    </tr>
+    <tr>
+        <td class="text-left" style="width: 20%">Verb: <code>GET</code></td>
+        <td class="text-left">Response code: <code>200 OK</code></td>
+    </tr>
+    <tr>
+        <td colspan="2">Returns the list of log files on a TaskManager.</td>
+    </tr>
+    <tr>
+        <td colspan="2">Path parameters</td>
+    </tr>
+    <tr>
+        <td colspan="2">
+            <ul>
+                <li><code>taskmanagerid</code> - 32-character hexadecimal string that identifies a task manager.</li>
+            </ul>
+        </td>
+    </tr>
+    <tr>
+        <td colspan="2">
+            <button data-toggle="collapse" data-target="#1220778126">Request</button>
+            <div id="1220778127" class="collapse">
+          <pre>
+            <code>
+{}            </code>
+          </pre>
+            </div>
+        </td>
+    </tr>
+    <tr>
+        <td colspan="2">
+            <button data-toggle="collapse" data-target="#1531506487">Response</button>
+            <div id="1531506488" class="collapse">
+          <pre>
+            <code>
+{
+  "type" : "object",
+  "id" : "urn:jsonschema:org:apache:flink:runtime:rest:messages:taskmanager:LogsInfo",
+  "properties" : {
+    "logs" : {
+      "type" : "array",
+      "items" : {
+        "type" : "object",
+        "id" : "urn:jsonschema:org:apache:flink:runtime:rest:messages:taskmanager:LogInfo",
+        "properties" : {
+          "name" : {
+            "type" : "string"
+          },
+          "size" : {
+            "type" : "long"
+          }
+        }
+      }
+    }
+  }
+}            </code>
+          </pre>
+            </div>
+        </td>
+    </tr>
+    </tbody>
+</table>
+<table class="table table-bordered">
+    <tbody>
+    <tr>
+        <td class="text-left" colspan="2"><h5><strong>/taskmanagers/:taskmanagerid/logs/:filename</strong></h5></td>
+    </tr>
+    <tr>
+        <td class="text-left" style="width: 20%">Verb: <code>GET</code></td>
+        <td class="text-left">Response code: <code>200 OK</code></td>
+    </tr>
+    <tr>
+        <td colspan="2">Provides access to get task manager file.</td>
+    </tr>
+    <tr>
+        <td colspan="2">Path parameters</td>
+    </tr>
+    <tr>
+        <td colspan="2">
+            <ul>
+                <li><code>taskmanagerid</code> - 32-character hexadecimal string that identifies a task manager.</li>
+                <li><code>filename</code> - string that identifies a file's name in task manager's log dir.</li>
```

Comment:
These descriptions should be generated by java code of `getDescription`. However, they seems not the same as current descriptions.